staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -337,6 +337,7 @@ type CustomResourceDefinitionCondition struct {
 // CustomResourceDefinitionStatus indicates the state of the CustomResourceDefinition
 type CustomResourceDefinitionStatus struct {
 	// Conditions indicate state for particular aspects of a CustomResourceDefinition
+	// +optional
 	Conditions []CustomResourceDefinitionCondition `json:"conditions" protobuf:"bytes,1,opt,name=conditions"`
```

Comment:
shouldn't this also set omitempty?

Reply:
> I'm going to withdraw omitempty from this pull, since there could be existing clients expecting .status.conditions: null from apiserver

just saw that comment, and the reasoning makes sense to me. nevermind.